hackathon_site/dashboard/frontend/src/components/dashboard/EditTeam/EditTeam.tsx

```diff
@@ -63,8 +75,16 @@ const TeamChangeForm = ({ canChangeTeam }: TeamModalProps) => {
                             color={"primary"}
                             className={styles.teamButton}
                             variant="contained"
-                            disabled={!canChangeTeam}
-                            onClick={() => alert(`The new teamcode is: ${value}`)}
+                            disabled={
+                                !canChangeTeam
+                                    ? true
+                                    : isJoinTeamLoading
+                                    ? true
+                                    : value == ""
+                                    ? true
+                                    : value == teamCode
+                            }
+                            onClick={handleJoinTeam}
```

Comment:
you should use formik here to get the value of the team in the text field instead of the React useState hook

Reply:
done!